spec/models/user_spec.rb

```diff
@@ -287,7 +287,7 @@
     it 'generates a secret 16 characters long' do
```

Comment:
this spec title is now wrong

```suggestion
    it 'generates a secret 32 characters long' do
```